### ISSUE TITLE
Survey evaluation processes among accessibility conformance evaluators

### ISSUE THREAD

**Issue by hidde** [w3c/wai-wcag-em#27]
It would be helpful to talk to different people who to accessibility conformance evaluations to find out if there are common or effective practices that are currently missing from WCAG-EM.

What questions should we ask them?
